contracts/tokenbridge/arbitrum/L2AtomicTokenBridgeFactory.sol

```diff
@@ -119,6 +119,10 @@ contract L2AtomicTokenBridgeFactory {
             Create2.deploy(0, _getL2Salt(OrbitSalts.L2_ROUTER_LOGIC), _creationCodeFor(runtimeCode));
         ProxyAdmin(proxyAdmin).upgrade(ITransparentUpgradeableProxy(canonicalRouter), routerLogic);
 
+        // init logic contract with dummy values
+        address dead = address(0x000000000000000000000000000000000000dEaD);
+        L2GatewayRouter(routerLogic).initialize(dead, dead);
```

Comment:
I think we should include a comment as to why we're doing these initializations

Reply:
I've added a more descriptive comment to 'ADDRESS_DEAD' definition - https://github.com/OffchainLabs/token-bridge-contracts/pull/54/commits/127c1f3d301a173a64dc27040df394a10938dbb3